charts/bandstand-cron-job/templates/pvc.yaml

```diff
@@ -1,17 +1,18 @@
-{{- if (.Values.volume).persistent }}
-{{- $relName := .Release.Name -}}
-{{- if .Values.nameSuffix }}
-{{- $relName = print .Release.Name "-" .Values.nameSuffix }}
-{{- end }}
+{{- if and .Values.s3Volumes (gt (len .Values.s3Volumes) 0) }}
+{{- range $volume := .Values.s3Volumes }}
+{{- $relName := printf "%s-%s" .Release.Name $volume.name }}
```

Comment:
Again confusing variable name. Not even sure you need a variable here, as it is only used once. Could just do it inline below